nixos/modules/flyingcircus/tests/default.nix

```diff
@@ -21,6 +21,11 @@
       inherit system;
     });
 
+  nodejs7 = hydraJob
+    (import ./nodejs7.nix {
+      inherit system;
+    });
+
```

Comment:
We don't need 3 nearly identical files. Refactor nodejs tests to a structure similar to mysql tests (see below).